public/base-locales/pl.json

```diff
@@ -1,50 +1,50 @@
 {
-  "login": "Login",
+  "login": "Zaloguj się",
   "click_once": "Kliknij 'Autoryzuj' tylko raz",
   "save": "Zapisz",
   "reset": "Zresestuj",
   "close": "Zamknij",
-  "reset_filters": "Zresetuj filtry",
+  "reset_filters": "Reset ustawień",
   "pokemon_filters": "Ustawienia filtrów Pokemonów",
   "gyms_filters": "Ustawienia filtrów gymów",
   "pokestops_filters": "Ustawienia filtrów Pokestopów",
   "nests_filters": "Ustawienia filtrów nestów",
   "help": "Pomoc",
   "apply_to_all": "Zastosuj do wszystkich",
-  "disable_all": "Wyłącz wszystkie",
-  "enable_all": "Włącz wszystkie",
-  "enabled": "Włączony",
+  "disable_all": "Dezaktywuj wszystkie",
+  "enable_all": "Aktywuj wszystkie",
+  "enabled": "Aktywuj",
   "legacy": "Legacy",
   "id": "ID",
   "location": "Lokalizacja",
   "search_pokemon": "Szukaj Pokemonów",
-  "search_nests": "Szukaj nestów",
-  "search_gyms": "Szukaj gymów",
-  "search_pokestops": "Szukaj Pokestopów",
-  "sm": "SM",
-  "md": "MD",
-  "lg": "LG",
+  "search_nests": "Szukaj nestujących Pokemonów",
+  "search_gyms": "Szukaj Gymów, Raidów i Raidbossów",
+  "search_pokestops": "Szukaj Inwazji, Lure i nagród z Questów",
+  "sm": "S",
+  "md": "M",
+  "lg": "L",
   "xl": "XL",
-  "min": "Min",
-  "max": "Max",
-  "set_size": "Ustaw rozmiar ikon",
-  "slot_selection": "Wybór szczeliny",
+  "min": "Min.",
+  "max": "Max.",
+  "set_size": "Rozmiar ikon",
+  "slot_selection": "Rozmiary Gymów per slot",
```

Comment:
I would make it `Rozmiary Gymów` without that `per slot`

Reply:
Context window is big enough to have `per slot` also - it makes it clear enough